client/reader/stream/index.jsx

```diff
@@ -468,7 +468,11 @@ class ReaderStream extends Component {
 			let sidebarContent = <ReaderSidebarFollowedSites path={ path } />;
 			if ( 'a8c' === streamType || 'p2' === streamType ) {
 				sidebarContent = (
-					<ReaderSidebarOrganizations organizations={ this.props.organizations } path={ path } />
+					<ReaderSidebarOrganizations
+						organizations={ this.props.organizations }
+						path={ path }
+						translate={ translate }
```

Comment:
While this will work, usually we use a "higher order component" to inject the `translate` prop automatically, avoiding the need to manually provide this prop for every component.

If you take a look a the `reader-sidebar-followed-sites/index.jsx` in this PR, you can see how this works:

```
import { localize } from 'i18n-calypso';

...

)( localize( ReaderSidebarFollowedSites ) );
```

You should be able to add the same to `reader-sidebar-organizations/index.jsx`. Then the `localize` function will automatically provide the translate function to `this.props` within the component, so you don't need to specify it here.

Reply:
Agreed here. We generally shouldn't be passing `translate()` as a prop from component to component; rather we should either use `useTranslate()` (for functional components) or the `localize()` HoC (for class components) to inject that prop there.